### PR TITLE
THRD-6

### PR DESCRIPTION
**Pull request by F1iper**

### CODE REVIEW

File: src/main/java/com/thehrdivision/entities/Employee.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    @OneToMany(mappedBy = "employee")
+    private Set<Payroll> payrolls;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<EmployeeSkills> employeeSkills;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<JobAssigments> jobAssigments;
+
+    // TODO: 19.03.2022 Employee <--> Benefit relation
```
Comment (a-dubaj) [inline src/main/java/com/thehrdivision/entities/Employee.java:50]: If we have another entity please create a task in JIRA, and delete this comment.

File: src/main/java/com/thehrdivision/entities/Payroll.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@Entity
+@Getter
+@Setter
+public class Payroll {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private LocalDate date; // date of what ?
```
Comment (F1iper) [inline src/main/java/com/thehrdivision/entities/Payroll.java:19]: date of payroll generation

File: src/main/java/com/thehrdivision/entities/Payroll.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    private String report; // ?
```
Comment (F1iper) [inline src/main/java/com/thehrdivision/entities/Payroll.java:20]: Auto-generated name of payroll.

Comment (a-dubaj) [inline src/main/java/com/thehrdivision/entities/Payroll.java:20]: For me `LocalDate date` stands for the date of generating the payroll. 
`String report` is the name for the generated report, like: 
`Payroll for Jan Kowalski <date >, number`

**Review by F1iper — COMMENTED** [review]
Resolved problem with date in Payroll entity -> comments included.

**Review by a-dubaj — APPROVED** [review]
Looks ok to me 👍

**Pull request merged**